cfme/tests/webui/test_splitter.py

```diff
@@ -18,45 +21,35 @@
 
 
 LOCATIONS = [
-    "control_explorer", "automate_explorer", "automate_customization", MyService,
-    "services_catalogs", "services_workloads", CustomReport, ComputeRate, Instance, Vm,
-    ISODatastore, Server, Datastore, ConfigManager, "utilization", "bottlenecks",
-    "infrastructure_networking"]
+    # Worksloads missing
+    # Bottlenecks missing
+    # infra_networking missing
+    (Server, 'ControlExplorer'), (Server, 'AutomateExplorer'), (Server, 'AutomateCustomization'),
+    (MyService, 'All'), (ServiceCatalogs, 'All'), (CustomReport, 'All'), (ComputeRate, 'All'),
+    (Instance, 'All'), (Vm, 'VMsOnly'), (ISODatastore, 'All'), (Server, 'Configuration'),
+    (Datastore, 'All'), (ConfigManager, 'All'), (Server, 'Utilization')
+]
 
 
 pytestmark = [pytest.mark.parametrize("location", LOCATIONS), pytest.mark.uncollectif(lambda
```

Comment:
for test readability it might help to put 2 parameter names into that parametrize and use them in the test function and in the navigate calls